features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimestampPosition.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.messages.impl.timeline.components
+
+enum class TimestampPosition {
+    /**
+     * Timestamp should be rendered above the timeline event content (eg. image).
+     */
+    Above,
```

Comment:
Since we have `Above` and `Below` and they're not opposites (otherwise the 'below' one would be behind the contents) I'd change this to `Overlayed` so there's no confusion. WDYT?

Reply:
Yes, I agree, from a 3D perspective `Below` is not the correct term, I'll take your proposal